charts/eodag-server/templates/deployment.yaml

```diff
@@ -144,6 +144,20 @@ spec:
             - name: EODAG_LOGGING
               value: {{ .Values.logLevel | quote }}
             {{- end }}
+            {{- if .Values.otel.enabled }}
+            {{- if .Values.otel.endpoint }}
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: {{ .Values.otel.endpoint | quote }}
+            {{- end }}
+            {{- if .Values.otel.timeout }}
+            - name: OTEL_EXPORTER_OTLP_TIMEOUT
+              value: {{ .Values.otel.timeout | quote }}
+            {{- end }}
+            {{- if .Values.otel.interval }}
+            - name: OTEL_METRIC_EXPORT_INTERVAL
+              value: {{ .Values.otel.interval | quote }}
+            {{- end }}
```

Comment:
I think we can simplify that and require OTEL_EXPORTER_OTLP_ENDPOINT, OTEL_EXPORTER_OTLP_TIMEOUT and OTEL_METRIC_EXPORT_INTERVAL if otel is enabled. No need to verify parameters individually.